### PR TITLE
Navigation: convert Light DOM LESS to SCSS

### PR DESCRIPTION
**Pull request by BY00565233**
JIRA: SURF-1891

### LGTM's
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/scss/components/navigation/_index.scss [hunk 1]
```diff
@@ -42,8 +39,8 @@
 
     &[aria-expanded="true"] {
       .toggle-icon {
-        .icon-transform();
-        transform+_: scaleY(-1);
+        transform: translateY(-50%);
+        transform: scaleY(-1);
```
Comment (CITguy) [inline src/scss/components/navigation/_index.scss]: Combine these two properties with the functions in the same order in the value.

```css
transform: translateY(-50%) scaleY(-1);
```

Comment (BY00565233) [reply src/scss/components/navigation/_index.scss]: Okay done and push the code into git Repository.

File: src/scss/components/navigation/_index.scss [hunk 2]
```diff
@@ -19,20 +19,17 @@
...
+      color: $gray-0;
     }
 
-    .icon-transform {
-      transform+_: translateY(-50%);
-    }
 
     .toggle-icon {
-      .icon-transform();
+      transform: translateY(-50%);
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss]: @BY00565233, please alphabetize.

Add after `top` property:

```less

  right: 0.75rem;
  top: 50%;
  transform: translateY(-50%);
}

```

Comment (BY00565233) [reply src/scss/components/navigation/_index.scss]: Changes has done.

**Pull request merged**